File: RB101_Programming_Foundations/Small Probems/Easy9/uppercase_check.rb
# Input : string
# output : return true if all alphabetic characters in the string are true, else false. Ignore non-alphabetic chracters

def uppercase?(string)
  string.upcase == string 
end

puts uppercase?('t') == false
puts uppercase?('T') == true
puts uppercase?('Four Score') == false
puts uppercase?('FOUR SCORE') == true
puts uppercase?('4SCORE!') == true
puts uppercase?('') == true

File: RB101_Programming_Foundations/small_problems/Easy9/Python/uppercase_check.py
# Write a method that takes a string argument, and returns true if all of the alphabetic characters inside the string are uppercase, false otherwise. Characters that are not alphabetic should be ignored.


'''
Input: string

Output: bool true/false if all chars in string is uppercase

Rules


Data structure / Algorithms
    if i make everything to uppercase and it is equal to input string, then every character is uppercase, so return true
    else return false


'''

def uppercase(string):
    return True if string.upper() == string else False

print(uppercase('t') == False)
print(uppercase('T') == True)
print(uppercase('Four Score') == False)
print(uppercase('FOUR SCORE') == True)
print(uppercase('4SCORE!') == True)
print(uppercase('') == True)